traffic_ops/traffic_ops_golang/invalidationjobs/invalidationjobs.go

```diff
@@ -42,52 +42,40 @@ import (
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/tenant"
 )
 
+// Deprecated, only to be used with versions below 4.0
 type InvalidationJob struct {
 	api.APIInfoImpl `json:"-"`
 	tc.InvalidationJob
 }
 
-const readQuery = `
-SELECT job.id,
-       keyword,
-       parameters,
-       asset_url,
-       start_time,
-       u.username AS createdBy,
-       ds.xml_id AS dsId
-FROM job
-JOIN tm_user u ON job.job_user = u.id
-JOIN deliveryservice ds  ON job.job_deliveryservice = ds.id
-`
+type InvalidationJobV40 struct {
+	api.APIInfoImpl `json:"-"`
+	tc.InvalidationJobV40
+}
```

Comment:
Are we using the CRUDer? I thought we wanted to move away from that as a project in favor of middlewares and expanding `APIInfo` structure capabilities?